psqlgraph/edge.py

```diff
@@ -104,6 +104,8 @@ def to_json(self):
         return {
             'src_id': self.src_id,
             'dst_id': self.dst_id,
+            'src_label': self.src.label,
```

Comment:
given that an association proxy won't be set on node creation, would it reasonable to (a) assert that `self.src is not None` and/or (b) sync it from the database?